src/assets/js/storedata.js

```diff
@@ -88,6 +88,25 @@ const parkeyevent = (event) => {
     removelist(id);
   }
 };
+const updatestatus = (id) => {
+  if (istoredlist()) {
+    let lists = JSON.parse(istoredlist());
+    lists = lists.map((list) => {
+      if (list.id === id) {
+        list.completed = !list.completed;
+      }
+      return list;
+    });
+    storelists(lists);
+  }
+};
+const showcheckbox = (event) => {
+  const parent = event.target.parentElement;
+  parent.classList.toggle('show-unchecked');
+  parent.classList.toggle('show-checked');
+  updatestatus(parent.id);
+};
+
```

Comment:
- [ ] As mentioned in the project requirements you should add a new JavaScript file and import it as a module and it will contain methods related to the status updates (`completed`: `true `/ `false`). So this method should be in their own file and you should import it, kindly do it 🙃 😉